src/Model/GoodiesManager.php

```diff
@@ -18,4 +20,33 @@ public function selectAllGoodies(string $orderBy = '', string $direction = 'ASC'
 
         return $this->pdo->query($query)->fetchAll();
     }
+
+    public function insert(array $item): int
```

Comment:
Je chipote mais c'est le nom de la variable ça serait mieux de l'appeler goodie c'est un tableau de goodie que tu vas insérer en bdd 
```suggestion
    public function insert(array $goodies): int
```

Reply:
Pareil j'ai changé le nom